src/cron/CronRunner.js

```diff
@@ -0,0 +1,86 @@
+
+import fs from "fs";
+import path from "path";
+import Cron from "cron";
+import _ from "underscore";
+
+const fname = 'cronJobs.initialize()';
+const p = '../../../../crons';
+
+
+
+export default class CronRunner {
+
+  constructor(felony) {
+    /**
+     * @type Felony
+     */
+    this.felony = felony;
+  }
+
+  /**
+   * Loads the job or fails silently.
+   *
+   * @param file
+   * @return {Object|null}
+   */
+  async _load (file) {
+    try {
+      return await import(`${this.felony.appRootPath}/crons/${file}`);
+    }
+    catch (e) {
+      console.log(`${fname} error, could not load "${file}" cron.`);
+      console.log(e);
+    }
+  };
+
+  async initialize() {
+    /**
+     * If the server is started with
+     * node index.js http cron
+     * Crons will be loaded
+     */
+    if (!Felony.arguments.cron) {
+      return;
+    }
+    global._cron = {};
+
+    // Get all the created cron jobs in "crons" folder.
+      fs.readdir(path.resolve('./crons'), async (error, jobs) => {
```

Comment:
There is a method on kernel that is dedicated to do these kind of things and is compatible with every OS, use it instead.